gutenberg.php

```diff
@@ -181,15 +181,13 @@ function gutenberg_intercept_edit_post() {
 		return;
 	}
 
-	if ( isset( $_GET['post'] ) ) {
-		$post_ID = (int) $_GET['post'];
-		$post_id = $post_ID;
-	}
-
-	if ( empty( $post_id ) ) {
+	if ( empty( $_GET['post'] ) || ! is_numeric( $_GET['post'] ) ) {
```

Comment:
In retrospect, `?post=0.1` would pass this check and be coerced into a `0` int value, which is what the previous condition was guarding against. However, I'd guess this would make it as far as the `$post` assignment and fail anyways on the subsequent falsey-ness check. Does that sound right to you, and then a non-concern?

Reply:
Yes, but there is no wy that `0.1` could come from the database (`ID` is a `BIGINT`), and also, yes, this would still be some (invalid) edge case. `-1` would also pass, but in any case, the check for the post object further down will fail, and all is good, I'd say.